health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/SetupMicroplan.js

```
@@ -124,14 +124,19 @@ const SetupMicroplan = ({ hierarchyType, hierarchyData }) => {
   };
 
   const onSubmit = (formData) => {
+    debugger;
```

Biome

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)